yogstation/code/game/area/areas/centcom.dm

```diff
@@ -31,4 +31,22 @@
 	blob_allowed = FALSE
 	has_gravity = TRUE
 	noteleport = TRUE
-	flags_1 = NONE
\ No newline at end of file
+	atmos = FALSE
```

Comment:
I don’t think this does anything

Reply:
it apparently disables atmos in the area; according to the code comments